tests/parser-spec.ts

```diff
@@ -633,6 +633,28 @@ describe('parsing', function() {
         });
     });
 
+    describe('svelte', function() {
+        it('parses a svelte file without included files', function() {
+            const file = getFixturePath('svelte.svelte');
+            const comments = getComments(file, {
+                withInlineFiles: false,
+            });
+            comments.should.have.length(1);
+            verifyComment(comments[0], 'TODO', 2, 'Svelte script comment');
+        });
+
+        it('parses a vue file with just php', function() {
+            const file = getFixturePath('vue.vue');
```

Comment:
Should use the `svelte.svelte` fixture :smile:

Reply:
Bah, didn't see it loaded twice. Should be all better now.